exopy/app/headless/manifest.enaml

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2021 by Exopy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Headless plugin manifest.
+
+"""
+import time
+import os
+import json
+
+from enaml.workbench.api import PluginManifest, Extension
+from enaml.workbench.ui.api import ActionItem
+from enaml.workbench.core.api import Command
+
+from ...measurement.measurement import Measurement
+from ..api import AppStartup
+from ...measurement.hooks.api import PostExecutionHook
+
+from ..headless.autoclose import AutoClose
+
+PLUGIN_ID ='exopy.app.headless'
+
+
+# =============================================================================
+# --- Factories ---------------------------------------------------------------
+# =============================================================================
+
+def log_plugin_factory():
+    """ Factory function for the LogPlugin.
+
+    """
+    from .plugin import LogPlugin
+    return LogPlugin()
+
+
+# =============================================================================
+# --- Startup handler ---------------------------------------------------------
+# =============================================================================
+
+def start_headless_measurement(workbench, cmd_args):
+    """Start headless measurement system.
+
+    """
+    path = cmd_args.execute
+    if path:
+        meas_plugin = workbench.get_plugin('exopy.measurement')
+        meas, errors = Measurement.load(meas_plugin, path)
+        if errors:
+            print(errors)
+        meas_plugin.processor.continuous_processing = False
+        meas_plugin.processor._run_measurement(meas, headless=True)
+
+# =============================================================================
+# --- Descriptions ------------------------------------------------------------
+# =============================================================================
+
```

Comment:
Would be nice to have a description.

Reply:
I've improved the doc a bit but I don't really see the point of having a bunch of descriptions since this plugin doesn't define any commands